src/Wallabag/CoreBundle/Helper/ContentProxy.php

```diff
@@ -47,10 +48,25 @@ public function updateEntry(Entry $entry, $url)
             }
         }
 
+        /*
+         * Set unique ids on paragraphs to set comments on the right ones
+         */
+        if (!strcmp($html, strip_tags($html)) == 0) {
+            $html5 = new HTML5();
```

Comment:
> What if the html isn't html5?

It uses DomDocument all the same, I think. I didn't encounter compatibility issues on the few non-html5 websites I tested.